### DISCUSSION TITLE
Unable to Add GitHub Pro

### DISCUSSION THREAD

**Discussion by Rameshchandrapola** [community/community#49440] [GitHub Education]
I have got GitHub pro on my institute Email ID account (@rameshchandra8520). But I was doing past work on my personal GitHub account (This). Now, I want to add my personal GitHub account as Pro and remove the institute Github account. 
Unable to do so.

![image.png](https://github.zendesk.com/attachments/token/YZTlVnScXAr5sjBQjKm9u6JBh/?name=image.png)

Please help me out.

**Comment by samus-aran** [comment]
Hi @Rameshchandrapola 

It’s not possible to transfer Global Campus benefits over from one GitHub account to another.

However, you are able to merge your accounts, but the account with the benefits would be the main account. You can add your non-academic email from the previous account to the account containing the academic email. If you desire you can then rename this account once merged. For more details check out out the [merging of your personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) documentation.
